presentation/onboarding/src/main/java/com/hilingual/presentation/onboarding/OnboardingScreen.kt

```diff
@@ -168,20 +182,37 @@ private fun OnboardingScreen(
 
         HilingualButton(
             text = "가입하기",
-            onClick = { isBottomSheetVisible = true },
+            onClick = { isTermsSheetVisible = true },
             enableProvider = isNicknameValid,
             modifier = Modifier.padding(vertical = 12.dp)
```

Comment:
```suggestion
            modifier = Modifier.padding(vertical = 16.dp)
```
엇 피그마 살펴보다가 우연히 발견했는데.. 여기 하단 여백이 16dp인 것 같아요..!

+) 참고: 피그마 캡쳐본
<img width="1250" height="320" alt="image" src="https://github.com/user-attachments/assets/4ce56368-3f3e-43d9-bfeb-7b58e0b5c9a4" />

Reply:
분명 16이엇는데...진짜 뭐지...고마워요ㅎㅎ